hardware/raspberrypi/frequency-management.md

```diff
@@ -8,13 +8,13 @@ For Raspberry Pi 3 Model B+, the PCB technology has been changed to provide bett
 
 The Raspberry Pi 4 Model B continues with the same PCB technology as the Raspberry Pi 3B+ to help dissipate excess heat. There is currently no soft limit defined.
```

Comment:
Raspberry Pi 4 Model B -> Raspberry Pi 4B.

Also a reference to Raspberry Pi 3 model B+ in the paragraph above.

2nd paragraph of page: remove reference to firmware after 2016 - no longer necessary.

Also, last paragraph of page - change 'gencmd' to 'command'.